guava/src/com/google/common/escape/Escapers.java

```diff
@@ -18,9 +18,14 @@
 
 import com.google.common.annotations.Beta;
 import com.google.common.annotations.GwtCompatible;
+import com.google.common.escape.CharEscaper;
```

Comment:
Why are you adding imports (other than to type annotations) to this file? That seems incorrect - the only reason to add imports would be changing the behavior of the code - which, as a goal, you do not want to do.